Directory.Build.props

```diff
@@ -2,8 +2,8 @@
   <PropertyGroup>
     <RepositoryName>wpf</RepositoryName>
     <WindowsDesktopARM64Support>true</WindowsDesktopARM64Support>
-    <TargetFramework>net8.0</TargetFramework>
-    <TargetFrameworkVersion>8.0</TargetFrameworkVersion>
+    <TargetFramework>net9.0</TargetFramework>
```

Comment:
@singhashish-wpf, why can't you use `NetCurrent` and `NetPrevious` variables defined by arcade?